recipes/passwordless/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "passwordless" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a17f4fd7b8f8a32981f9abf8870dd19130e4d1f9c82337e87f624a894d1c9efa
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.8
+    - poetry-core
+    - pip
+  run:
+    - python >=3.8
+    - requests >=2.0,<3.0
+    - marshmallow >=3.0,<4.0
+    - python-dateutil >=2.0,<3.0
+
+test:
+  requires:
+    - pip
+  imports:
+    - passwordless
+  commands:
+    - pip check
+
+about:
+  home: https://bitwarden.com/products/passwordless
+  summary: 'The official Bitwarden Passwordless.dev Python library, for Python 3+.'
+  description: |
+    Your customers, users, and workforce want smooth and secure login experiences,
+    free from traditional passwords. Meet Bitwarden Passwordless.dev, the easiest
+    way to build passkeys.
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: LICENSE
+  doc_url: https://docs.passwordless.dev/guide
+  dev_url: https://docs.passwordless.dev/guide
```

Comment:
Maybe change the `dev_url` to https://github.com/bitwarden/passwordless-python

Reply:
@synapticarbors : Good idea! I have changed the dev_url.

Reply:
@conda-forge/help-python : Ready for a new review.